connector/sumconnector/config.go

```diff
@@ -24,3 +34,88 @@ type AttributeConfig struct {
 	Key          string `mapstructure:"key"`
 	DefaultValue any    `mapstructure:"default_value"`
 }
+
+func (c *Config) Validate() error {
+	for name, info := range c.Spans {
+		if name == "" {
+			return fmt.Errorf("spans: metric name missing")
```

Comment:
Do you think it would be valuable to get all of errors found and return them, rather than just the first? It might help someone debug their configuration faster, but may make the logic a bit harder to follow.

Reply:
Yeah that makes sense to me! Updated along with testing for multi errors